src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/enrich/util.ts

```diff
@@ -99,7 +99,7 @@ export const modeDescription = i18n.translate(
   }
 );
 
-export const modeSuggestions: SuggestionRawDefinition[] = ENRICH_MODES.map(
+export const modeSuggestions: SuggestionRawDefinition[] = ENRICH_MODES?.map(
```

Comment:
Why this change?

Reply:
A test was failing but I could not understand why tbh